react-native/rn-bounce/package.json

```diff
@@ -9,7 +9,7 @@
   "types": "./index.d.ts",
   "peerDependencies": {
     "react": "^17.0.0",
```

Comment:
good idea to do it for react as well?

Reply:
Well, reading the docs this carret range would allow for all versions lower than 18. If this is your intention you can leave it as is I suppose.